### DISCUSSION TITLE
Inverted layout problem

### DISCUSSION THREAD

**Discussion by wawadanon** [xournalpp/xournalpp#5949] [Q&A]
I just downloaded the app, but the whole layout is reversed. how do i fix this? 
<img width="960" alt="{265D50BC-0F2F-41F9-8EF9-A42E13ADDD8C}" src="https://github.com/user-attachments/assets/0fa30007-b44f-48a3-956a-6692f927a815">

**Comment by bhennion** [comment]
:-) For some reason, the app thinks your locale is Right-to-Left (like in many languages). Is your computer configured to use a language in which one usually writes from right to left?

**Reply by wawadanon** [reply]
yes though im not sure why. i changed everything to english but in some apps the configuration is still in another language 😢

**Reply by wawadanon** [reply]
oh wait nevermind. turns out there's a 'regional format' setting in addition to language 🤦‍♂️  it works now
